app/code/Magento/ConfigurableProduct/Model/ResourceModel/Product/BaseSelectProcessorInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\ConfigurableProduct\Model\ResourceModel\Product;
+
+use Magento\Framework\DB\Select;
+
+/**
+ * Interface BaseSelectProcessorInterface
+ * @api
```

Comment:
for patch releases, do not mark newly introduced interfaces as api

Reply:
fixed